guides/chalet_time_policy.md

```diff
@@ -54,64 +66,75 @@ Priorities for chalet time:
+Bids team will update the Scheduling team each week of bids people with chalet time can get involved with.
+
+Marketing team will maintain a visible backlog of tasks people with chalet time can do to help promote Made Tech. This will be posted in the [`#chalet-time-team`][1] Slack channel at the beginning of each week.
 
 #### 3. Hiring
-Scheduling team will give the Talent team coordinators access to the Kimble report of who has chalet to do extra hiring interviews or require training to do so
 
-Scheduling team will giving the Talent team coordinators access to the Kimble report of who’s has chalet 
+Scheduling team will give the Talent team coordinators access to the Kimble report of who has chalet to do extra hiring interviews or require training to do so.
+
+Scheduling team will giving the Talent team coordinators access to the Kimble report of who’s has chalet.
 
 #### 4. Research and development
 
-The R&D team will maintain a visible backlog of tasks people with chalet time can do either to help build products or research new opportunities. This will be posted in the [`#chalet-time-team`][1] Slack channel at the beginning of each week. 
+The R&D team will maintain a visible backlog of tasks people with chalet time can do either to help build products or research new opportunities. This will be posted in the [`#chalet-time-team`][1] Slack channel at the beginning of each week.
 
 ##### 5. Communities of practice
 
-Capability and Delivery Heads will maintain a [visible backlog of tasks](https://trello.com/b/taj8yvLP/capability-improvement-backlog) people with chalet time can undertake to improve their community of practice. This will be posted weekly in both [`#chalet-time-team`][1] and [`#cop-improvements-backlog`](https://madetechteam.slack.com/archives/C03BMF2E39S) Slack channels 
+Capability and Delivery Heads will maintain a [visible backlog of tasks](https://trello.com/b/taj8yvLP/capability-improvement-backlog) people with chalet time can undertake to improve their community of practice. This will be posted weekly in both [`#chalet-time-team`][1] and [`#cop-improvements-backlog`](https://madetechteam.slack.com/archives/C03BMF2E39S) Slack channels.
 
 ## Length of chalet time activities
-Chalet time activities must be able to deliver some value in small blocks of time: half day, single day or 3-5 days, eaning chalet time can add value if someone joins a client team at short notice. 
 
-Some activities may need more than a week but still be able to deliver business or personal value in increments of a half day, single day or 3-5 days.
+Chalet time activities must be able to deliver some value in small blocks of time: half day, single day or 3-5 days, meaning chalet time can add value if someone joins a client team at short notice.
```

Comment:
My only comment is to check that the dash in "3-5 days" should be a [figure dash](https://graphemica.com/%E2%80%92) (‒, `U+2012`) or an [en-dash](https://graphemica.com/%E2%80%93) (–, `U+2013`) rather than a [hyphen-minus](https://graphemica.com/-) (-, `U+002D`).

Apart from that, this whole PR looks good to me; good job! 🙂

Reply:
Wow, that is some good 👁️ work!
I've replaced both hyphen-dashes with en-dashes (`Opt+-` on mac).

Reply:
I did proof reading for a while at uni. It's a skill that's hard to turn off 😉

LGTM